Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1870,6 +1870,7 @@ a_Player:OpenWindow(Window);
 				Feed = { Params = "AddFood, AddSaturation", Return = "bool", Notes = "Tries to add the specified amounts to food level and food saturation level (only positive amounts expected). Returns true if player was hungry and the food was consumed, false if too satiated." },
 				FoodPoison = { Params = "NumTicks", Return = "", Notes = "Starts the food poisoning for the specified amount of ticks; if already foodpoisoned, sets FoodPoisonedTicksRemaining to the larger of the two" },
 				ForceSetSpeed = { Params = "{{Vector3d|Direction}}", Notes = "Forces the player to move to the given direction." },
+				Freeze = { Params = "{{Vector3d|Location}}", Return = "", Notes = "Prevent the player from moving and lock him into Location until unfreeze() is called" },
```

Comment:
Assuming SetPosition and `ClientHandle->SendPlayerPosition` work for any arbitary distance, then yes, he is teleported.